src/store/selectors.js

```diff
@@ -0,0 +1,19 @@
+import { createRegistrySelector } from '@wordpress/data';
+
+
+
+
+
+
+export const getBlocks = ( state ) => {
+	return state.present.blocks || [];
```

Comment:
I've updated this to allow for a default/fallback set of blocks, so that can reload blocks previously worked on;
 
```
export const getBlocks = ( state, defaultBlocks=[] ) => {
	return state.present.blocks || defaultBlocks;
}
```

```
  const blocks = useSelect((select) => select("getdavesbe").getBlocks(parse(storedBlocks)));
```

Reply:
Yep although grabbing for storage is a side effect so should probably be handled in a control action. Still working that one out.